bundles/org.openhab.binding.ferroamp/README.md

```diff
@@ -0,0 +1,187 @@
+# ferroamp Binding
+
+The ferroamp binding is used to get live data from ferroamp energy systems.
+
+The binding retrieve data from the different parts of the ferroamp energy system such as:
+
+- `Ehub`: EnergyHub Wall and EnergyHub XL.
+- `Sso`: Solar string optimizer.
+- `Eso`: Bidirectional DC/DC converter for connection of battery.
+- `Esm`: Energy Storage Module.
+ 
+
+## Supported Things
+
+This binding is compatible with EnergyHub Wall and EnergyHub XL, and use connection to your local Ehub via LAN or similar.
+Data and commands are sent using MQTT where the user connects to the MQTT broker residing on the ferroamp system.
+Contact ferroamp support to enable MQTT in the EnergyHub and to get Username and Password:
+https://ferroamp.com/om-ferroamp/
+
+## Discovery
+
+Discovery is not supported, as the system has a number of fixed parameters.
+
+## Thing Configuration
+
+Binding MQTT must be installed before binding ferroamp.
+
+Installing of binding ferroamp and configuration of it can be done in the UI.
+
+The following configuration-parameters are available for the `ferroamp` binding.
+
+| Name            | Type    | Description                           | Default | Required | Advanced |
+|-----------------|---------|---------------------------------------|---------|----------|----------|
+| hostname        | text    | Hostname or IP address of the device  | N/A     | yes      | no       |
+| username        | text    | Username to access the device         | N/A     | yes      | no       |
+| password        | text    | Password to access the device         | N/A     | yes      | no       |
+| hasBattery      | boolean | Is there a battery connected or not ? | N/A     | no       | yes      |
+
+
+## Channels
+
+The following channels are available for the ferroamp binding.
+
+| Channel Type ID                   | Item Type               | Read/Write | Description                              |
+|-----------------------------------|------------------------ |------------|------------------------------------------|
+| device-local-ehub-wloadconsq-l1   | Number:Energy           | R          |                                          |
```

Comment:
All channel type id's have the `device-local` prefix. Did you consider removing this prefix? As what is the use?